### PR TITLE
Alternatives to Capitalist Structures

### PR DESCRIPTION
**Pull request by Frijol**
For the discussion (and many other reading suggestions!) see https://github.com/datatogether/reading_datatogether/issues/41

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,9 +74,16 @@ If the levers of capitalism place it in opposition to just data practices, can w
 
 **Readings:**
 
-Coming soon! Readings should appear at least 1 month before the date of discussion.
-
-Please feel free to suggest & recommend readings on the [Issue](https://github.com/datatogether/reading_datatogether/issues/41) for this session!
+1. Kathi Weeks (2011): **The Problem with Work p5-8, 42-47, 51-57** Available at: [libcom.org](https://libcom.org/files/the-problem-with-work_-feminism-marxism-kathi-weeks.pdf)
+1. Hanna Hurr (2016): **Silvia Federici interviewed in Mask Magazine** Available at: [maskmagazine.com](http://www.maskmagazine.com/the-control-issue/struggle/interview-silvia-federici)
```
Comment (dcwalk) [inline README.md]: I think we usually show the full URL (e.g., "maskmagazine.com/the-control-issue/struggle/interview-silvia-federici"), I actually don't know why, I think a vestigial imagining as a potentially printed documents?

Comment (dcwalk) [inline README.md]:
```suggestion
1. Hanna Hurr (2016): **Silvia Federici interviewed in Mask Magazine** Available at: [maskmagazine.com/the-control-issue/struggle/interview-silvia-federici](http://www.maskmagazine.com/the-control-issue/struggle/interview-silvia-federici)
```

File: README.md [hunk 2]
```diff
@@ -74,9 +74,16 @@ If the levers of capitalism place it in opposition to just data practices, can w
...
+1. Cory Doctorow (2017): **Walkaway** (a novel)– [this excerpt](https://www.tor.com/2017/04/03/excerpts-cory-doctorow-walkaway-chapter-2/)
```
Comment (dcwalk) [inline README.md]:
```suggestion
1. Cory Doctorow (2017): **Walkaway** (a novel) [Excerpt]. Available at: [tor.com/2017/04/03/excerpts-cory-doctorow-walkaway-chapter-2/](https://www.tor.com/2017/04/03/excerpts-cory-doctorow-walkaway-chapter-2/)
```

File: README.md [hunk 3]
```diff
@@ -74,9 +74,16 @@ If the levers of capitalism place it in opposition to just data practices, can w
...
+1. Gibson-Graham, J.K. and E. Miller (2015): "**Economy as Ecological Livelihood**" Available at: [communityeconomies.org](http://www.communityeconomies.org/publications/chapters/economy-ecological-livelihood)
```
Comment (dcwalk) [inline README.md]:
```suggestion
1. Gibson-Graham, J.K. and E. Miller (2015): "**Economy as Ecological Livelihood**" Available at: [communityeconomies.org/publications/chapters/economy-ecological-livelihood](http://www.communityeconomies.org/publications/chapters/economy-ecological-livelihood)
```

File: README.md [hunk 4]
```diff
@@ -74,9 +74,16 @@ If the levers of capitalism place it in opposition to just data practices, can w
...
+1. Frase, Peter (2011): "**Four Futures**" Available at: [jacobinmag.com](https://www.jacobinmag.com/2011/12/four-futures/)
```
Comment (dcwalk) [inline README.md]:
```suggestion
1. Frase, Peter (2011): "**Four Futures**" Available at: [jacobinmag.com/2011/12/four-futures/](https://www.jacobinmag.com/2011/12/four-futures/)
```

File: README.md [hunk 5]
```diff
@@ -74,9 +74,16 @@ If the levers of capitalism place it in opposition to just data practices, can w
...
+1. Michael Johnson (2012): **The Cooperative Principles, the Common Good, and Solidarity** Available at: [geo.coop](http://www.geo.coop/story/cooperative-principles-common-good-and-solidarity)
```
Comment (dcwalk) [inline README.md]:
```suggestion
1. Michael Johnson (2012): **The Cooperative Principles, the Common Good, and Solidarity** Available at: [geo.coop/story/cooperative-principles-common-good-and-solidarity](http://www.geo.coop/story/cooperative-principles-common-good-and-solidarity)
```

**Comment by Frijol** [comment]
Thanks! (I figured out how to batch commit your suggestions halfway through 🙄 whoops– cool feature though!)

**Pull request merged**